### ISSUE TITLE
Allow failures on "latest" or "nightly"?

### ISSUE THREAD

**Issue by jayschwa** [simonbyrne/Appveyor.jl#12]
```
environment:
  matrix:
  - julia_version: 0.7
  - julia_version: 1
  - julia_version: nightly
```
```
# # Uncomment the following lines to allow failures on nightly julia
# # (tests will run but not make your overall status red)
# matrix:
#   allow_failures:
#   - julia_version: latest
```
Is `julia_version: latest` correct, or is that supposed to be `julia_version: nightly`?

**Comment by simonbyrne** [comment]
good catch!

**Comment by simonbyrne** [comment]
I had renamed "latest" to "nightly" to make it clearer and keep it consistent with travis, but missed this one.

Fixed by 195d111.
